hydra-node/test/Hydra/Chain/Direct/TimeHandleSpec.hs

```diff
@@ -21,10 +21,10 @@ spec = do
             pure $ res === slot
 
   it "should convert slot within latest/current era" $ do
-    let now = posixSecondsToUTCTime 13.4
+    let currentSlotNo = SlotNo 10
```

Comment:
The equivalent would be `13`, but this is fine as well .. I hope (can't recall this unit test in detail)